.github/workflows/conventional-commit-lint.yaml

```diff
@@ -0,0 +1,36 @@
+name: conventionalcommits.org
+
+on:
+  pull_request:
+    types: ['opened', 'edited', 'reopened', 'synchronize']
+
+jobs:
+  lint:
+    name: Validate PR title
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: read
+    steps:
+      - uses: amannn/action-semantic-pull-request@v6
+        with:
+          # Ensure the subject starts with an uppercase letter and does not include a Jira issue.
+          # Jira issues are supposed to be added at the end of the issue in brackets.
+          subjectPattern: ^(?!.*\b[A-Z]+-\d+\b)[A-Z].*
+          # Error explaining how to write a commit.
+          subjectPatternError: |
+            The subject "{subject}" found in the pull request title "{title}"
+            didn't match the configured pattern. Please ensure that the subject
+            starts with an uppercase character and that Jira issues are provided in 
+            brackets at the end of the commit in brackets, e.g. "feat: New feature (SC-12, SC-34)".
+          # Extends the base conventional commit RegEx with a convention to include Jira issues.
+          # Jira issues are provided optionally, though specified here to ensure in the subject pattern 
+          # that they aren't included, in the subject pattern.
+          # Expected PR title: chore(learning-plan): Updates dependencies (SC-1234, AI-2, PT-1)
+          headerPattern: '^(\w*)(?:\(([\w$.\-*/ ]*)\))?: (.*)(?: \(((?:SC|AI|PT)-\d+(?:, )?)+\))?$'
```

Comment:
```suggestion
          headerPattern: '^(\w*)(?:\(([\w$.\-*/ ]*)\))?: (.*?)(?: \(((?:(?:SC|AI|PT)-\d+(?:, )?)+)\))?$'
```

Reply:
Improves that the subject capture group is not greedy to match as few characters as posible.

And ensures that the jiraIssues capture group captures all issues not just the last one.